storybookjs/storybook
# UI: Fix keybindings on non-US keyboard layouts

Issue: #13296

## What I did

This fixes the `/` keybinding on Hungarian keyboards (and probably other). Besides that, I've taken the opportunity to refactor some of this logic, without introducing a whole library dependency.

## How to test

Test the `/` keybinding to focus the search input using `Shift+6` on a Hungarian keyboard layout.

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
